### PR TITLE
[fix-4933]: When tabbing and pressing enter over a filter, toggle filter.

### PR DESCRIPTION
**Pull request by OscarBakker**
Ticket: [SIG-4933](https://datapunt.atlassian.net/browse/SIG-4933)

## changes
currently on meldingenkaart, a user cant focus and keypress 'enter' to toggle this filter. With this pr it can.

## Signalen

Before opening a pull request, please ensure:

- Make sure your PR title follows naming conventions: [feat-1234]: name feature
- Double-check your branch is based on `main` and targets `main`
- Pull request has tests (we are going for 100% coverage!)
- Code is well-commented, linted and follows project conventions
- Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: src/signals/IncidentMap/components/FilterPanel/FilterPanel.test.tsx [hunk 1]
```diff
@@ -97,6 +96,29 @@ describe('FilterPanel', () => {
...
+
+    userEvent.click(checkbox)
+
+    expect(
+      mockSetFilters.mock.calls[1][0].find(
+        (filter: Filter) => filter.name === testCategory
+      ).filterActive
+    ).toBe(false)
+  })
+
```
Comment (vdegraaf) [inline src/signals/IncidentMap/components/FilterPanel/FilterPanel.test.tsx:121]: Maybe good to check whether the filterActive was true initially.

File: src/signals/IncidentMap/components/FilterPanel/FilterCategory.tsx [hunk 2]
```diff
@@ -22,7 +22,15 @@ export const FilterCategory = ({
       data-testid={text}
       id={text}
       checked={selected}
-      onChange={onToggleCategory}
+      onChange={() => {
+        onToggleCategory()
+      }}
```
Comment (vdegraaf) [inline src/signals/IncidentMap/components/FilterPanel/FilterCategory.tsx:27]: Why did you change this? It does the same `onChange={onToggleCategory}` right?

Comment (OscarBakker) [reply src/signals/IncidentMap/components/FilterPanel/FilterCategory.tsx:27]: oh yeah, will change it back.

File: src/signals/IncidentMap/components/FilterPanel/FilterPanel.test.tsx (hunk 1 above)
Comment (OscarBakker) [reply src/signals/IncidentMap/components/FilterPanel/FilterPanel.test.tsx:121]: yeah, well its true by default, but true, can add it to assure.

**Review by vdegraaf — APPROVED** [review]
Approved once feedback is incorporated
